.coderabbit.yml

```
@@ -0,0 +1,120 @@
+          Generate React component documentation with:
+          - @component tag and component description
+          - @param for all props with detailed descriptions
+          - @example showing typical usage patterns
+          - Accessibility notes where relevant
+
+# Custom rules for this TypeScript/React CLI project
+rules:
+  typescript:
+    strict_mode: true
+    prefer_type_annotations: true
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)